walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,69 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredAndOptionalIgnored() {
```

Comment:
Here and elsewhere: I didn't really understand what this test was testing just by looking at the name. It's generally very good practices to have test names be really clear on what behavior they are testing.

"considered" is too ambiguous here to make it clear what exactly the test expects.

Test names should be in third person and describe the behavior and circumstances if applicable.

E.g. "DoesXWhenY", not "TakesXIntoConsideration"

Good read: https://drive.google.com/file/d/0B_-S6UdTLV5pNnhTRnFzYnRHenM/view

Something like "PrefersTimeSlotWhereOptionalAttendeesCanAttend" "IgnoresOptionalAttendeesIfNoSuitableTimeSlotFound" might be clearer.

Sadly even the existing tests in this file are not following best practices.